src/ctap/storage.rs

```diff
@@ -541,9 +543,18 @@ impl PersistentStore {
         Ok(())
     }
 
+    /// Returns whether the PIN needs to be changed before its next usage.
+    pub fn has_force_pin_change(&self) -> Result<bool, Ctap2StatusCode> {
+        match self.store.find(key::FORCE_PIN_CHANGE)? {
+            None => Ok(false),
+            Some(value) if value.len() == 1 && value[0] == 1 => Ok(true),
```

Comment:
If the value is constant, why not have a value of length zero? Then it's just about the presence of the entry. Is it because it might change in the future?

Reply:
Zero length is okay, too. This was just supposed to be an explicit `true`. I can change it.

I was thinking if I should piggy-back it onto some other key like PIN related keys, since they need to be changed anyway and we save a delete. But went with the explicit and clean solution instead. It's a rare enough event that we don't have to care I think.

Reply:
The advantage of zero-length is:
1. It only costs 1 word of flash instead of 2.
2. There are less invariants.

Reply:
Good to know for size on flash! Changed.